### PR TITLE
Trev8939/RGBRenderer

### PR DESCRIPTION
**Pull request by TADraeseke**
Adds a sample which demonstrates use of the RGBRenderer.

### CODE REVIEW

File: rgb-renderer/src/main/res/layout/dialog_box.xml [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+            app:layout_constraintBottom_toBottomOf="@+id/max_blue_value_text_view"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/max_blue_value_text_view"/>
+
+    <!-- PercentClip -->
+    <TextView
+            android:id="@+id/percent_clip_min_value_text_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="16dp"
```
Comment (doneill) [inline rgb-renderer/src/main/res/layout/dialog_box.xml]: Redundant as you defined `layout_marginStart` below

File: rgb-renderer/src/main/res/layout/dialog_box.xml [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+            android:layout_marginEnd="16dp"
+            app:layout_constraintBottom_toBottomOf="@+id/percent_clip_min_value_text_view"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/percent_clip_min_value_text_view"
+            app:layout_constraintVertical_bias="0.0"/>
+    <TextView
+            android:id="@+id/curr_percent_clip_max_text_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginRight="16dp"
```
Comment (doneill) [inline rgb-renderer/src/main/res/layout/dialog_box.xml]: Redundant as you defined `layout_marginEnd` below

File: rgb-renderer/src/main/java/com/esri/arcgisruntime/sample/rgbrenderer/ParametersDialogFragment.java [hunk 3]
```diff
@@ -0,0 +1,484 @@
...
+    mCurrPercentClipMaxTextView = (TextView) dialogView.findViewById(R.id.curr_percent_clip_max_text_view);
+    //update seek bar positions with current PercentClip
+    updateSeekBar(mPercentClipMinSeekBar, mPercentClipMin, mCurrPercentClipMinTextView);
+    updateSeekBar(mPercentClipMaxSeekBar, mPercentClipMax, mCurrPercentClipMaxTextView);
+    // standard deviation ui elements
+    mStdDevTextView = (TextView) dialogView.findViewById(R.id.std_dev_text_view);
+    mStdDevSeekBar = (SeekBar) dialogView.findViewById(R.id.std_dev_seek_bar);
+    mStdDevSeekBar.setMax(3);
+    mCurrStdDevTextView = (TextView) dialogView.findViewById(R.id.curr_std_dev_text_view);
+    //update seel bar position with current StandardDeviation
```
Comment (doneill) [inline rgb-renderer/src/main/java/com/esri/arcgisruntime/sample/rgbrenderer/ParametersDialogFragment.java]: Typo in `seel`

**Comment by TADraeseke** [comment]
Thanks @doneill! All comments addressed. Back to you!

**Comment by doneill** [comment]
@TADraeseke Looks good to merge!

**Comment by TADraeseke** [comment]
Cheers Dan, thanks!

**Pull request merged**